pkg/occlient/occlient.go

```diff
@@ -3069,6 +3069,18 @@ func (c *Client) IsVolumeAnEmptyDir(volumeMountName string, dc *appsv1.Deploymen
 	return false
 }
 
+// IsVolumeAnConfigMap returns true if the volume is an ConfigMap, false if not
+func (c *Client) IsVolumeAnConfigMap(volumeMountName string, dc *appsv1.DeploymentConfig) bool {
+	for _, volume := range dc.Spec.Template.Spec.Volumes {
+		if volume.Name == volumeMountName {
+			if volume.ConfigMap != nil {
+				return true
```

Comment:
Wouldn't `if volume.Name == volumeMountName && volume.ConfigMap != nil` be better? My only reasoning for saying it "better" is that it is a one-liner and reduces a nested `if`. :)